apps/api/src/database/migrations/1745495895857-InitialSeed.ts

```
@@ -0,0 +1,333 @@
+
+    try {
+      for (const providerData of masterProvidersData) {
+        await queryRunner.manager
+          .createQueryBuilder()
+          .insert()
+          .into('notify_master_providers')
+          .values(providerData)
+          .execute();
+      }
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)